e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_1_spec.js

```diff
@@ -441,6 +441,38 @@ describe('Keyboard Shortcuts', () => {
         cy.get('#moreDmModal').should('not.exist');
     });
 
+    it('MM-T4452 - CTRL/CMD+SHIFT+. Expand or collapse RHS', () => {
+        // # Post a message and open RHS
+        const message = `hello${Date.now()}`;
+        cy.postMessage(message);
+        cy.getLastPostId().then((postId) => {
+            // # Mouseover the post and click post comment icon.
+            cy.clickPostCommentIcon(postId);
+        });
```

Comment:
```suggestion
        // # Mouseover the post and click post comment icon.
        cy.clickPostCommentIcon();  
```
Without params the above command execute on the last post itself. 
see :
https://github.com/mattermost/mattermost-webapp/blob/eeb471bec051f0a515eea705c6041643a9c571dd/e2e/cypress/support/ui_commands.js#L340-L343